### PR TITLE
Fix and enable release blocker plugin

### PR DESCRIPTION
**Pull request by xpivarc**
/cc @brianmcarey @rmohr

### CODE REVIEW

File: github/ci/prow-deploy/kustom/base/configs/current/plugins/plugins.yaml [hunk 1]
```diff
@@ -427,11 +427,13 @@ external_plugins:
     events:
       - pull_request
   - name: cherrypicker
+  - name: release-blocker
```
Comment (brianmcarey) [inline github/ci/prow-deploy/kustom/base/configs/current/plugins/plugins.yaml:430]: Do we need to build a new image and update the deployment manifest before adding this back in? 

https://github.com/kubevirt/project-infra/blob/f96e43d0b8d6aec90ef48b8b02b49b3266d9be74/github/ci/prow-deploy/kustom/base/manifests/local/release-blocker_deployment.yaml#L20

**Comment by xpivarc** [comment]
/hold
@brianmcarey good point. Thanks!

File: external-plugins/release-blocker/main.go [hunk 2]
```diff
@@ -98,7 +98,7 @@ func main() {
 
 	server := &Server{
 		tokenGenerator: secretAgent.GetTokenGenerator(o.webhookSecretFile),
-		botName:        botUserData.Name,
+		botName:        botUserData.Login,
```
Comment (rmohr) [inline external-plugins/release-blocker/main.go:101]: big impact :)

**Review by rmohr — COMMENTED** [review]
Like @brianmcarey pointed out, we need a new image first. Fix looks great.

**Comment by xpivarc** [comment]
@brianmcarey Can I ask you for the image push? (I don't have access)

**Comment by xpivarc** [comment]
/hold 
I don't see the error locally. Will have a look.

**Comment by brianmcarey** [comment]
> @brianmcarey Can I ask you for the image push? (I don't have access)

I've built and pushed the updated release blocker image - `quay.io/kubevirtci/release-blocker:v20220815-ab4bb7e6`

**Comment by xpivarc** [comment]
/hold cancel
@brianmcarey Thanks!

**Review by brianmcarey — COMMENTED** [review]
/lgtm

**Pull request merged**